### PR TITLE
Sec area xfmr

### PR DESCRIPTION
**Pull request by AAndersn**
Remove boundary devices from lower level areas:
* remove secondary transformers from secondary area
* remove switches from switch switch area

### CODE REVIEW

File: cimgraph/loaders/sparql/rc4_2021/switch_phase.py [hunk 1]
```diff
@@ -45,25 +44,22 @@ class type and object mRID
...
                   bind(concat(str(?meas_id),",",strafter(str(?meas_cls),"CIM100#")) as ?Measurement).}
 
-        OPTIONAL {?eq cim:SwitchPhase.closed ?closed>
-        OPTIONAL {?eq cim:SwitchPhase.normalOpen ?normalOpen>
+        OPTIONAL {?eq cim:SwitchPhase.closed ?closed.}
+        OPTIONAL {?eq cim:SwitchPhase.normalOpen ?normalOpen.}
         OPTIONAL {?eq cim:SwitchPhase.phaseSide1 ?phs1.
                  bind(strafter(str(?phs1),"SinglePhaseKind.") as ?phaseSide1)}
-        OPTIONAL {?eq <cim:SwitchPhase.phaseSide2 ?ph2.
+        OPTIONAL {?eq cim:SwitchPhase.phaseSide1 ?phs2.
```
Comment (afisher1) [inline cimgraph/loaders/sparql/rc4_2021/switch_phase.py:58]: shouldn't this be phaseSide2?

**Review by afisher1 — CHANGES_REQUESTED** [review]
@AAndersn There was a change in the sparql query in switch_phase.py that I think isn't right. I left a comment at the line in question.

**Comment by afisher1** [comment]
@AAnderson I don't see any code changes in this PR that would perform the removal of boundary devices from the lower level areas and testing my app with these changes confirms that boundary devices are still being located in lower level areas.

File: cimgraph/loaders/sparql/rc4_2021/switch_phase.py (hunk 1 above)
Comment (afisher1) [reply cimgraph/loaders/sparql/rc4_2021/switch_phase.py]: After talking with @AAndersn it should be phaseSide2 and has been resolved

**Review by afisher1 — CHANGES_REQUESTED** [review]
@AAndersn looking at all the code changes I see nothing in here that would cause the removal of boundary devices from existing in the lower level distributed areas. Testing with the distributed ybus service still shows the transformer is still showing up in the secondary area for the ieee13nodeckt.

**Comment by AAndersn** [comment]
Andy, there seems to be something wrong with git on my server. There should be a commit with changes to SecondaryArea.py and SwitchArea py. My history shows it was pushed but I can't find it on GitHub

**Comment by afisher1** [comment]
@AAndersn , I just checked the history on those files on sec_area_xfmr branch. The last commits for cimgraph/models/switch_area.py and cimgraph/models/secondary_area.py were on February 4. So somehow your changes didn't get pushed from your local repo somehow. if you still having issues with git, can you try emailing me the files?

**Review by afisher1 — APPROVED** [review]
This has all typo fixes and boundary device issues present and working as expected.

**Comment by craig8** [comment]
Fixed #11

**Pull request merged**